### PR TITLE
More accurate refresh rates (mHz)

### PR DESCRIPTION
**Pull request by ascent12**

### CODE REVIEW

File: include/wlr/wayland.h [hunk 1]
```diff
@@ -10,8 +10,8 @@ struct wlr_output_mode_state;
 struct wlr_output_mode {
 	struct wlr_output_mode_state *state;
 	uint32_t flags; // enum wl_output_mode
-	int32_t width, height;
-	int32_t refresh; // mHz
+	uint32_t width, height;
+	uint32_t refresh; // mHz
```
Comment (SirCmpwn) [inline include/wlr/wayland.h]: This is actually taken straight from the wayland protocol, which uses signed integers. Why? I have no clue.

Comment (ascent12) [inline include/wlr/wayland.h]: Yeah, it makes absolutely no sense for them to be negative, so I changed them. They can be changed back to signed though; it wouldn't hurt anything.

Comment (SirCmpwn) [inline include/wlr/wayland.h]: Please do. Also address the merge conflicts that were introduced by merging your other PR.

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**